README.md

```diff
@@ -40,6 +40,13 @@ Build fonts to edit TrueType instructions in VTT (placed next to the UFO)
 $ make vtt
 ```
 
+When working on the fonts, "normalize" them before you check them in. Tools
+tend to format the underlying files however they want, leading to large,
+pointless differences to what's in the repository.
+```
+$ make normalize
```

Comment:
Can't see this rule in the resulting `Makefile` anymore.  Is this textual description accurate + relevant?